src/Jackett.Common/Definitions/exitorrent-org.yml

```diff
@@ -104,7 +104,7 @@ search:
       args: [" +(?:19|20)\\d{2} *$", ""]
 
   rows:
-    selector: table[class="text"][border="0"][cellspacing="0"][cellpadding="4"] > tbody > tr:has(a[href^="details.php?id="]):has(td[class*="tableBody1"])
+    selector: table[class="text"][border="0"][cellspacing="0"][cellpadding="4"] > tbody > tr:has(a[href^="details.php?id="]):has(td[class*="tableBody1"], td[class*="tableBody1B"])
```

Comment:
if the last selector does the job, do we need the first selector anymore?

Reply:
On the site there are multiple different td-s that has class tableBody1 or tableBody1B and this is the only differentiator from others. They are used interchangeably so when I get rid of one there may be some results missing. Maybe You could do it better but I found it work OK.